CycloneDX/Services/ProjectAssetsFileService.cs

```diff
@@ -48,27 +47,25 @@ public HashSet<NugetPackage> GetNugetPackages(string projectFilePath, string pro
             if (_fileSystem.File.Exists(projectAssetsFilePath))
             {
                 var assetFileReader = _assetFileReaderFactory();
-                string jsonContent = assetFileReader.ReadAllText(projectAssetsFilePath);
-                JsonDocument assetFileObject = _assetJsonObject.Parse(jsonContent);
-                // get all direct nuget dependencies of the project
-                JsonElement frameworksProperties = assetFileObject.RootElement.GetProperty("project").GetProperty("frameworks");
-
                 var assetsFile = assetFileReader.Read(projectAssetsFilePath);
 
                 foreach (var targetRuntime in assetsFile.Targets)
                 {
                     var runtimePackages = new HashSet<NugetPackage>();
+                    var targetFramework = assetsFile.PackageSpec.GetTargetFramework(targetRuntime.TargetFramework);
```

Comment:
Before the PR, `targetRuntime.Name` was used to resolve the framework. This sadly includes the `RuntimeIdentifier` which is not part of the package spec and would lead to exceptions (my initial report).

The `LockFile` already has support for resolving this without the need of manually trying to map this, making `TargetFrameworkToAlias` obsolete.